### PR TITLE
Update core

### PR DESCRIPTION
**Pull request by Pacman99**
Most of the changes were in digga

### CODE REVIEW

**Comment by blaggacao** [comment]
Should we wait for @gytis-ivaskevicius to cut a release for `fup`?

**Comment by Pacman99** [comment]
> Should we wait for @gytis-ivaskevicius to cut a release for fup?

Since fup input is locked in digga, I personally don't think thats a blocker. If we extensively test digga with a certain fup version then it doesn't really matter if we're using staging or master

**Comment by Pacman99** [comment]
> Should we timely move forward with this, too, and then continue to polish from a fresh master / core?

Yes lets do that

File: doc/concepts/suites.md [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
+profiles. For good examples, check out the suites defined in the community branch.
 
-In the future, we will use suites as a mechanism for deploying various machine
-types which don't depend on hardware, such as vm's and containers.
+`suites` are a special case of an `importable` which get passed as a special
+argument (one that can be use in an `imports` line) to your hosts.
 
-They are defined with the `suites` argument in either `home` or `nixos` namespace.
+They are defined with the `suites` argument in either the `home` or `nixos` namespace.
 Suites should be passed as a function that take profiles as an argument.
```
Comment (blaggacao) [inline doc/concepts/suites.md:9]: I think this is a left over.

**Review by blaggacao — APPROVED** [review]

File: flake.nix [hunk 2]
```diff
@@ -5,7 +5,7 @@
     {
       nixos.url = "nixpkgs/nixos-unstable";
       latest.url = "nixpkgs";
-      digga.url = "github:divnix/digga";
+      digga.url = "github:divnix/digga/develop";
```
Comment (blaggacao) [inline flake.nix:8]: I guess that should be master, now.

**Review by blaggacao — COMMENTED** [review]
`digga:develop` -> `digga:master` !?

**Review by blaggacao — APPROVED** [review]

**Pull request merged**